frontend/app/views/spree/checkout/_payment.html.erb

```diff
@@ -59,7 +59,15 @@
     <p class='field' data-hook='coupon_code'>
       <%= form.label :coupon_code %><br />
       <%= form.text_field :coupon_code %>
+      <button type="button" class="button" id="coupon-code-apply-button">
+        <%= Spree.t(:apply_code) %>
+      </button>
     </p>
+    <div id='coupon_status'
+      data-success-class="success"
```

Comment:
They imply that they are overwriting default behaviour. If we want to document it both the changelog and the JS file make very good candidates for that.